ui_testing/testcases/basic_tests/test03_orders.py

```diff
@@ -112,6 +112,90 @@ def test003_cancel_button_edit_departments(self, save):
                                                                                       order_departments))
             self.assertEqual(current_departments, order_departments)
 
+    @parameterized.expand(['save_btn', 'cancel'])
+    def test006_update_material_type(self, save):
+        """
+        New: Orders: Edit material type: Make sure that user able to change material type and related test plan &
+        article.
+
+        LIMS-4281
+        LIMS-4281
+        :return:
+        """
+        self.order_page.get_random_order()
+        order_url = self.base_selenium.get_url()
+        self.base_selenium.LOGGER.info(' + order_url : {}'.format(order_url))
+        current_material_type = self.order_page.get_material_type()
+        self.order_page.set_material_type()
```

Comment:
changing material type raises a pop-up and you didn't handle it.
![image](https://user-images.githubusercontent.com/17981089/61034098-ec024500-a3c4-11e9-92fc-318bd186b26c.png)

Reply:
I already put  ( self.order_page.confirm_popup(force=True) )

Reply:
u put it in a wrong order. Execute your test case and I will see that it doesnot interact with the pop-up

Reply:
This is should put before get ( done)